pkg/gcptarget/matcher_test.go

```diff
@@ -0,0 +1,87 @@
+package gcptarget
+
+import (
+	"errors"
+	"testing"
+)
+
+func TestMatch(t *testing.T) {
+	tests := []struct {
+		name    string
+		include []string
+		exclude []string
+		review  interface{}
+		want    bool
+		wantErr error
+	}{
+		{
+			name:    "include **",
+			include: []string{"**"},
+			review: map[string]interface{}{
+				"ancestry_path": "abc/def",
+			},
+			want: true,
+		},
+		{
+			name:    "include with *",
+			include: []string{"*/def", "*/abc"},
+			review: map[string]interface{}{
+				"ancestry_path": "abc/def",
+			},
+			want: true,
+		},
+		{
+			name:    "include not match",
+			include: []string{"*/abc"},
+			review: map[string]interface{}{
+				"ancestry_path": "abc/def",
+			},
+			want: false,
+		},
+		{
+			name:    "exclude",
+			include: []string{"**"},
+			exclude: []string{"abc/*"},
+			review: map[string]interface{}{
+				"ancestry_path": "abc/def",
+			},
+			want: false,
+		},
+		{
+			name:    "exclude not match",
```

Comment:
Would it make sense to have tests for cases like:

```
			include: []string{"**"},
			exclude: []string{"abc/**"},
```

Or is that getting too in the weeds / tested elsewhere?

Reply:
Added